### ISSUE TITLE
Use of Eval Causing Problems in Prettier-Atom

### ISSUE THREAD

**Issue by robwise** [prettier/prettier#1895]
Atom is angry at v1.4 because [using eval](https://github.com/prettier/prettier/blob/ba86d8ea652fe302817add3d55e8c8244593817b/parser.js#L13) breaks its content security policy (CSP). https://github.com/prettier/prettier-atom/issues/179

```
EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".
```

<br />

Usually, the workaround is [this](https://github.com/atom/loophole#eval-loophole-), but for some reason, using that loophole breaks node's ability to resolve the require statements properly:

```
Error: Cannot find module './src/parser-babylon'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/src/module-cache.js:350:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.parse (/Users/Rob/github/prettier-atom/node_modules/prettier/index.js:12064:36)
    at formatWithCursor (/Users/Rob/github/prettier-atom/node_modules/prettier/index.js:12267:22)
    at format (/Users/Rob/github/prettier-atom/node_modules/prettier/index.js:12307:10)
    at formatWithShebang (/Users/Rob/github/prettier-atom/node_modules/prettier/index.js:12465:12)
    at Object.format (/Users/Rob/github/prettier-atom/node_modules/prettier/index.js:12481:12)
    at /Users/Rob/github/prettier-atom/dist/executePrettier/executePrettierOnBufferRange.js:23:29
```

I can't figure out a way around this, any ideas?

**Comment by vjeux** [comment]
I'm using eval to hide it from rollup and webpack. I can replace it afterwards. I'll fix it for 1.4.1 sorry about that!

**Comment by robwise** [comment]
Oh awesome, no worries!!

**Comment by vjeux** [comment]
With https://github.com/prettier/prettier/pull/1925, when I do the release (in a few minutes) it shouldn't have eval anymore :)

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!

**Comment by robwise** [comment]
Confirmed! Awesome!
